libcloud/storage/drivers/azure_blobs.py

```diff
@@ -217,15 +221,34 @@ def __init__(self, key, secret=None, secure=True, host=None, port=None,
                                                       port=port, **kwargs)
 
     def _ex_connection_class_kwargs(self):
```

Comment:
Some unit tests for this method would be good.

Reply:
Added tests for the Azurite path in f9313bd6 and for the new GovCloud path in 9071060d.